modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -37,18 +37,20 @@ def get_appointments(start_date, end_date)
         end
 
         def put_cancel_appointment(params)
-          cancel_reasons = get_facility_cancel_reasons(params[:facilityId])
+          facility_id = Mobile::V0::Appointment.toggle_non_prod_id!(params[:facilityId])
+
+          cancel_reasons = get_facility_cancel_reasons(facility_id)
           cancel_reason = extract_valid_reason(cancel_reasons.map(&:number))
 
           raise Common::Exceptions::BackendServiceException, 'MOBL_404_cancel_reason_not_found' if cancel_reason.nil?
 
           put_params = {
-            appointment_time: DateTime.parse(params[:appointmentTime]).strftime('%m/%d/%Y %H:%M:%S'),
+            appointment_time: params[:appointmentTime].strftime('%m/%d/%Y %H:%M:%S'),
```

Comment:
Will params[:appointmentTime] just be a string? I'm guessing with the removal of DateTime.parse it won't be but just wanted to double check.

Reply:
ya it's `DateTime` now, it get's parsed further up the chain